ptvsd/wrapper.py

```diff
@@ -1491,6 +1491,8 @@ def on_stackTrace(self, request, args):
                 # This means the stack was requested before the
                 # thread was suspended
                 xframes = []
+            else:
+                xframes = list(xframes)
```

Comment:
This should be done in the suspend handler. So it will be resolved for all current and future uses of the stack frames list.

Reply:
fixed